udata_hydra_csvapi/query.py

```diff
@@ -18,15 +10,20 @@ async def get_resource(session: ClientSession, resource_id: str, columns: list):
     async with session.get(url) as res:
         record = await res.json()
         if not res.ok:
-            raise QueryException(res.status, record)
+            handle_exception(res.status, "Database error", record, resource_id)
         if not record:
             raise web.HTTPNotFound()
         return record[0]
 
 
-async def get_resource_data(session: ClientSession, resource: dict, query_string: str):
-    async with session.get(f"{config.PG_RST_URL}/{resource['parsing_table']}?{query_string}") as res:
+async def get_resource_data(session: ClientSession, resource: dict, sql_query: str):
+    headers = {"Prefer": "count=exact"}
+    url = f"{config.PG_RST_URL}/{resource['parsing_table']}?{sql_query}"
+    async with session.get(url, headers=headers) as res:
         if not res.ok:
-            raise QueryException(res.status, await res.json())
-        async for chunk in res.content.iter_chunked(1024):
-            yield chunk
+            handle_exception(
+                res.status, "Database error", await res.json(), resource["id"]
```

Comment:
`resource["id"]` doesn't seem to exist. When running a failing query ([example](http://localhost:8005/api/resources/27d469ff-9908-4b7e-a2e0-9439bb38a395/data/?ips_voie_pro__greater=100&page=600&page_size=10)), the error doesn't show correctly